config/kcp/apiexport_has.yaml

```diff
@@ -7,6 +7,6 @@ metadata:
   name: has
 spec:
   latestResourceSchemas:
```

Comment:
But I wonder if this is actually HAS' responsibility? Here are my arguments:
- All the team's controllers would be eventually working on kcp in unison, so would it not be the primary controller's responsibility to generate `APIResourceSchema` for their respective APIs? Same way I would assume build-service wouldnt be generating an `APIResourceSchema` for `Component` even when they have a controller for `Component`
- When we're using HAS controllers in dev/staging; we need to explicitly install the binding, snapshot, etc CRDs; otherwise our controller will error out. So I am wondering if this is any different to how the aforementioned teams like build-service, managed-gitops, etc dont really do install `Component` CRDs even if they have a controller for it 

🤔

Reply:
You're correct that it's not our responsibility, but I was making a note just that this will complicate testing on KCP until `ThingPermissionClaim` support is in KCP, and the APIResourceSchemas / APIExports for the Snapshot & Binding CRDs are generated by GitOps.

Reply:
Until then, we'll need to manually generate the APIResourceSchemas and add it to our APIExports before testing.

Reply:
ok, do we just copy paste the CR spec in the field there for these new CRs? 🤔